hudi-spark-datasource/hudi-spark/src/test/resources/sql-statements.sql

```diff
@@ -63,10 +63,10 @@ select id, name, price, cast(dt as string) from h0_p;
 # CREATE TABLE
 
 create table h1 (
-  id bigint,
+  id int,
   name string,
   price double,
-  ts bigint
+  ts int
```

Comment:
Why do `id` and `ts` have to be integer type?  Let's keep them in long type and fix MERGE INTO statements only, i.e., other DML statements should not be affected.

Reply:
If changing `id` column values involve more changes, let's keep `ts` column as `long` type.